Source/Core/Core/Core.cpp

```diff
@@ -278,6 +278,10 @@ void Stop()  // - Hammertime!
 #if defined(__LIBUSB__)
   GCAdapter::ResetRumble();
 #endif
+#if defined(CIFACE_USE_XINPUT) || defined(CIFACE_USE_DINPUT)
+  for (int i = 0; i != 4; ++i)
```

Comment:
This can be placed within its own static local function so it's not duplicated in another spot.

e.g.

```cpp
static void ResetRumble()
{
#if defined(__LIBUSB__)
  GCAdapter::ResetRumble();
#endif
#if defined(CIFACE_USE_XINPUT) || defined(CIFACE_USE_DINPUT)
  for (int i = 0; i < 4; ++i)
    Pad::ResetRumble(i);
#endif
}
```